frontend/src/Components/Modals/ModalModifyGame.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Modal, ModalHeader, ModalBody, Button } from 'shards-react';
+
+const ModalModifyGame = ({ open, setOpen, title, action, id, handleGridUpdate, handleOverwriteClick, setUserGamesUpdated }) => {
+  const history = useHistory();
+  const [accepted, setAccepted] = useState(false);
+  const { getAccessTokenSilently, user } = useAuth0();
+
+  useEffect(() => {
+    function cleanUp() {
+      setOpen(false);
+      setAccepted(false);
+      history.push('/');
+    }
+
+    function cleanUpAfterDelete() {
+      setOpen(false);
+      setAccepted(false);
+      setUserGamesUpdated(true);
+    }
+
+    const saveGameLoad = async () => {
+      if (accepted){
+        try {
+          const accessToken = await getAccessTokenSilently({
+            audience: process.env.REACT_APP_AUTH0_AUDIENCE,
+            scope: process.env.REACT_APP_AUTH0_SCOPE,
+          });
+    
+          const saveLoaded = await fetch(`http://localhost:3001/sudoku/${user.sub}?saveGame=${id}`, {

```

Comment:
Ensure that the fetch url changes in a later PR when the backend is hosted on Heroku.